examples/spring/src/main/kotlin/com/expediagroup/graphql/examples/dataloaders/CompanyService.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Expedia, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.expediagroup.graphql.examples.dataloaders
+
+import com.expediagroup.graphql.examples.model.Company
+import org.springframework.stereotype.Component
+
+@Component("companyServiceBean")
```

Comment:
whats the point of renaming from default name to `companyServiceBean` (i.e. adding bean suffix)?

Reply:
The application fails to start the examples if this is not included